src/Teams/Controllers/MultipleAnswerQuestionController.cs

```diff
@@ -37,5 +39,47 @@ public IActionResult MultipleAnswerQuestionForm(string id, int[] answers)
             };
             return View(question);
         }
+
+        public IActionResult EditMultipleAnswerQuestion(Guid id)
+        {
+            var question = new MultipleAnswerQuestionViewModel()
+            {
+                SourceQuestion = questionRepository.PickById(id),
+            };
+            return View(question);
+        }
+
+        public IActionResult AddMultipleAnswerQuestion()
+        {
+            return View();
+        }
+
+        [HttpPost]
+        public IActionResult AddMultipleAnswerQuestion([FromBody] MultipleQuestionAddModel fromAjax)
+        {
+            List<MultipleAnswerQuestionOption> allAnswers = new List<MultipleAnswerQuestionOption>();
+            allAnswers = fromAjax.TextArray.Select((value, index) => new MultipleAnswerQuestionOption(value, fromAjax.CheckboxValueArray[index])).ToList();
+
+            questionRepository.AddQuestion(new MultipleAnswerQuestion(fromAjax.QuestionText, allAnswers));
+            _db.SaveChanges();
```

Comment:
Since you created repository( starting from ef core some guys says that we can omit repository pattern) Add method SaveChangesAsync to repository and make calls using repository.